examples/time_series/goes_xrs_nrt_data.py

```diff
@@ -64,9 +67,30 @@
 goes_ts = ts.TimeSeries(goes_data, meta, units, source="xrs")
 
 ###############################################################################
-# Finally, we can plot the timeseries.
+# Now we will download the latest flare event information
+
+flare_events = pd.read_json('https://services.swpc.noaa.gov/json/goes/primary/xray-flares-7-day.json')
```

Comment:
So NOAA just keep a list of flares with some basic info for 7 days?

Reply:
yes and there is also a latest flare file with just the last one.